packages/console/src/components/UnsavedChangesAlertModal/index.tsx

```diff
@@ -68,12 +68,10 @@ const UnsavedChangesAlertModal = ({ hasUnsavedChanges }: Props) => {
   return (
     <ConfirmModal
       isOpen={displayAlert}
-      confirmButtonType="primary"
```

Comment:
do we need to set confirmButtonType danger?

Reply:
It is 'danger' in `ConfirmModal` by default.